From caef30c607a1880b2f7ff9dc00accc2e791cee00 Mon Sep 17 00:00:00 2001
From: Mike Drob <mdrob@cloudera.com>
Date: Tue, 9 May 2017 06:55:42 -0800
Subject: [PATCH 332/362] Revert "HBASE-17761: Test
 TestRemoveRegionMetrics.testMoveRegion fails
 intermittently because of race condition"

This reverts commit 39a35ceec30c1f233b3fc72bd6e6b8368c8fb141.

Change-Id: I590e6d3231b11817c1bbb4b4e220750fc67be093
---
 .../apache/hadoop/hbase/HBaseTestingUtility.java   |   24 --------------------
 1 file changed, 24 deletions(-)

diff --git a/hbase-server/src/test/java/org/apache/hadoop/hbase/HBaseTestingUtility.java b/hbase-server/src/test/java/org/apache/hadoop/hbase/HBaseTestingUtility.java
index a82a622..e46fa37 100644
--- a/hbase-server/src/test/java/org/apache/hadoop/hbase/HBaseTestingUtility.java
+++ b/hbase-server/src/test/java/org/apache/hadoop/hbase/HBaseTestingUtility.java
@@ -3444,30 +3444,6 @@ public class HBaseTestingUtility extends HBaseCommonTestingUtility {
   }
 
   /**
-   * Move region to destination server and wait till region is completely moved and online
-   *
-   * @param destRegion region to move
-   * @param destServer destination server of the region
-   * @throws InterruptedException
-   * @throws IOException
-   */
-  public void moveRegionAndWait(HRegionInfo destRegion, ServerName destServer)
-      throws InterruptedException, IOException {
-    HMaster master = getMiniHBaseCluster().getMaster();
-    getHBaseAdmin().move(destRegion.getEncodedNameAsBytes(),
-        Bytes.toBytes(destServer.getServerName()));
-    while (true) {
-      ServerName serverName = master.getAssignmentManager().getRegionStates()
-          .getRegionServerOfRegion(destRegion);
-      if (serverName != null && serverName.equals(destServer)) {
-        assertRegionOnServer(destRegion, serverName, 200);
-        break;
-      }
-      Thread.sleep(10);
-    }
-  }
-
-  /**
    * Wait until all regions for a table in hbase:meta have a non-empty
    * info:server, up to 60 seconds. This means all regions have been deployed,
    * master has been informed and updated hbase:meta with the regions deployed
-- 
1.7.9.5

